### PR TITLE
Blog App: Create model

### PR DESCRIPTION
**Pull request by Caren-Koroeny**
<h1> Hello :wave:  reviewer(s)</h1>

> In this milestone I was able to:

:heavy_minus_sign: Create a new rails application

:heavy_minus_sign:  Connect `PostgreSQL` to my project

 :heavy_minus_sign:Apply my knowledge of migration in the milestone for instance used 'rails generate migration CreateUsers'

 :heavy_minus_sign: Check and fix all milestone offences

<h2> :brain: Knowledge Gained :deciduous_tree: </h2>

:white_check_mark: Install Ruby on Rails framework.

:white_check_mark: Use Ruby Gems as a software packages system.

:white_check_mark: Understand what ORM is.

:white_check_mark: Use database migration files to maintain database schema.

<h2> Thank you for taking the time to review my project :pray: </h2>

### CODE REVIEW

**Review by 1GORDON — COMMENTED** [review]
##
## Hi @Caren-Koroeny ✋

Good job so far!

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

### This project will be marked as Invalid and won't affect the number of your weekly reviews.

- [ ] Kindly add the correct [readme template](https://github.com/microverseinc/readme-template) and customize it accordingly. Remember to remove the Microverse logo, Add the Future Features section and 1 - 3 features to be added.
- [ ] Also fix the broken MIT license link to make your readme file more professional.

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: db/migrate/20230608001833_add_user_ref_to_like.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class AddUserRefToLike < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :likes, :user, null: false, foreign_key: true
```
Comment (DuaneDave) [inline db/migrate/20230608001833_add_user_ref_to_like.rb]: If you check the diagram provided, you'd see authorId on most of the models and that should serve as the foreign key field in the table. So in order to implement the ERD correctly, consider using this syntax here:

```
`t.references :author, foreign_key: { to_table: 'users' }`
```

File: db/migrate/20230608001504_add_user_ref_to_comment.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class AddUserRefToComment < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :comments, :user, null: false, foreign_key: true
```
Comment (DuaneDave) [inline db/migrate/20230608001504_add_user_ref_to_comment.rb]: If you check the diagram provided, you'd see authorId on most of the models and that should serve as the foreign key field in the table. So in order to implement the ERD correctly, consider using this syntax here:

```
`t.references :author, foreign_key: { to_table: 'users' }`
```

File: db/migrate/20230608001258_add_user_reference_to_post.rb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+class AddUserReferenceToPost < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :posts, :user, null: false, foreign_key: true
```
Comment (DuaneDave) [inline db/migrate/20230608001258_add_user_reference_to_post.rb]: If you check the diagram provided, you'd see authorId on most of the models and that should serve as the foreign key field in the table. So in order to implement the ERD correctly, consider using this syntax here:

```
`t.references :author, foreign_key: { to_table: 'users' }`
```

**Review by DuaneDave — CHANGES_REQUESTED** [review]
Hi @Caren-Koroeny 👋 , 

Good job so far on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ Good job setting up the migrations correctly

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @Caren-Koroeny👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**